app/utils/wordcloud/wordcloud_demodata_load.py

```diff
@@ -0,0 +1,91 @@
+'''
+Notebook used to load the demo data into a hosted Postgres database.
+Demo data is 167 transcribed anonymous student story submissions.
+Database will be hosted on AWS to be used as demo data
+for the word cloud feature for the parent dashboard.
+'''
+
+import psycopg2
+import os
+from dotenv import load_dotenv
+import sqlite3
+from sqlite3 import Error
+from zipfile import ZipFile
+from psycopg2.extras import execute_values
+
+load_dotenv()
+
+def create_connection():
+    """elephantsql connection"""
+    #connect to ElephantSQL-hosted PostgreSQL
+    DB_NAME = os.getenv("DB_NAME", default="OOPS")
+    DB_USER = os.getenv("DB_USER", default="OOPS")
+    DB_PASSWORD = os.getenv("DB_PASSWORD", default="OOPS")
+    DB_HOST = os.getenv("DB_HOST", default="OOPS")
+
+    pg_connection = psycopg2.connect(dbname=DB_NAME, user=DB_USER, password=DB_PASSWORD, host=DB_HOST)
+    
+    return pg_connection
+
+def insert_data(conn, story):
+    '''
+    Insert the demo data into the table
+    '''
+    sql = f'''INSERT INTO stories (story)
+             VALUES (%s)'''
+    cur = conn.cursor()
+    cur.execute(sql, story)
+    conn.commit()
+    return cur.lastrowid
+
+def ingest_zip():
+    '''opens the zip file of the submissions'''
+    z = ZipFile(r'C:\Users\temsy\Downloads\story transcripts.zip', 'r')
```

Comment:
Could you turn this into a reference to a file in the repo? Or maybe to a file on the RDS?

Reply:
For example:
self.all_pages = pd.read_csv("./data/37_stories_meta.csv")

Reply:
I will look into this